03_launch_mgmt_cluster.sh

```diff
@@ -500,21 +500,15 @@ create_clouds_yaml
 if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
   start_management_cluster
   kubectl create namespace metal3
-fi
 
-if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
   patch_clusterctl
   launch_cluster_api_provider_metal3
-fi
 
-if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
     if [ "${CAPM3_VERSION}" != "v1alpha4" ]; then
         launch_baremetal_operator
     fi
     launch_ironic
-fi
```

Comment:
I think we can improve a bit further 
```shell
if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
  start_management_cluster
  kubectl create namespace metal3

  patch_clusterctl
  launch_cluster_api_provider_metal3

    if [ "${CAPM3_VERSION}" == "v1alpha4" ]; then
        BMO_NAME_PREFIX="${NAMEPREFIX}-baremetal-operator"
    else
        BMO_NAME_PREFIX="${NAMEPREFIX}"
        launch_baremetal_operator
    fi
    launch_ironic

```

Reply:
Good point, thanks! Will update :slightly_smiling_face: